test/04_openst/01_pricer_rule.js

```diff
@@ -0,0 +1,57 @@
+// 'use strict';
```

Comment:
We already have 01_direct_transfer.js. We can rename 01_pricer_rule.js => 02_pricer_rule.js.